documentation/seed-node.md

```diff
@@ -0,0 +1,255 @@
+
+Begin by downloading geth and switching to the vulcanize/rpc_statediffing branch
+
+`go get github.com/ethereum/go-ethereum`
+
+`cd $GOPATH/src/github.com/ethereum/go-ethereum`
+
+`git remote add vulcanize https://github.com/vulcanize/go-ethereum.git`
+
+`git fetch vulcanize`
+
+`git checkout -b rpc_statediffing vulcanize/rpc_statediffing`
+
+Now, install this fork of geth (make sure any old versions have been uninstalled/binaries removed first)
+
+`make geth`
+
+And run the output binary with statediffing turned on:
+
+`cd $GOPATH/src/github.com/ethereum/go-ethereum/build/bin`
+
+`./geth --statediff --statediff.streamblock --ws --syncmode=full`
+
+Note: other CLI options- statediff specific ones included- can be explored with `./geth help`
+
+The output from geth should mention that it is `Starting statediff service` and block synchronization should begin shortly thereafter.
+Note that until it receives a subscriber, the statediffing process does essentially nothing. Once a subscription is received, this 
+will be indicated in the output. 
+
+Also in the output will be the websocket url and ipc paths that we will use to subscribe to the statediffing process.
+The default ws url is "ws://127.0.0.1:8546" and the default ipcPath- on Darwin systems only- is "Users/user/Library/Ethereum/geth.ipc"
+
+### Vulcanizedb
+
+There are two commands to choose from:
+ 
+#### syncAndPublish
+ 
+`syncAndPublih` performs the functions of the seed node- syncing data from Geth, converting them to IPLDs,
+publishing those IPLDs to IPFS, and creating a local Postgres index to relate their CIDS to useful metadata. 
+
+Usage:
+
+`./vulcanizedb syncAndPublish --config=<config_file.toml>`
+
+The config file for the `syncAndPublish` command looks very similar to the basic config file
+```toml
+[database]
+    name     = "vulcanize_demo"
+    hostname = "localhost"
+    port     = 5432
+
+[client]
+    ipcPath  = "ws://127.0.0.1:8546"
+    ipfsPath = "/Users/user/.ipfs"
+```
+
+With an additional field, `client.ipcPath`, that is either the ws url or the ipc path that Geth has exposed (the url and path output
+when the geth sync was started), and `client.ipfsPath` which is the path the ipfs datastore directory.
+
+#### syncPublishScreenAndServe
+
+`syncPublishScreenAndServe` does everythin th at `syncAndPublish` does, plut it opens up an RPC server which exposes
+an endpoint to allow transformers to subscribe to subsets of the sync-and-published data that are relevant to thier transformations
+
+Usage:
+
+`./vulcanizedb syncPublishScreenAndServe --config=<config_file.toml>`
+
+The config file for the `syncPublishScreenAndServe` command has two additional fields and looks like:
+
+```toml
+[database]
+    name     = "vulcanize_demo"
+    hostname = "localhost"
+    port     = 5432
+
+[client]
+    ipcPath  = "ws://127.0.0.1:8546"
+    ipfsPath = "/Users/user/.ipfs"
+
+[server]
+    ipcPath = "/Users/user/.vulcanize/vulcanize.ipc"
+    wsEndpoint = "127.0.0.1:2019"
+```
```

Comment:
How will the user know what to configure these values as?

Reply:
These are user defined so they can configure them to be whatever they like so long as they are valid paths/endpoints. Although transformers that wish to subscribe to the seed node will need to know these values in order to subscribe, so we probably want to standardize this somehow...